src/Api/Teams.php

```diff
@@ -9,13 +9,66 @@
 
 namespace PrivatePackagist\ApiClient\Api;
 
+use PrivatePackagist\ApiClient\TeamPermissions;
+
 class Teams extends AbstractApi
 {
     public function all()
     {
         return $this->get('/teams/');
     }
 
+    public function create(string $name, TeamPermissions $permissions): array
+    {
+        $parameters = [
+            'name' => $name,
+            'permissions' => [
+                'canEditTeamPackages' => (bool) $permissions->canEditTeamPackages,
+                'canAddPackages' => (bool) $permissions->canAddPackages,
+                'canCreateSubrepositories' => (bool) $permissions->canCreateSubrepositories,
+                'canViewVendorCustomers' => (bool) $permissions->canViewVendorCustomers,
+                'canManageVendorCustomers' => (bool) $permissions->canManageVendorCustomers,
+            ],
+        ];
+
+        return $this->post('/teams/', $parameters);
+    }
+
+    public function edit($teamId, string $name, TeamPermissions $permissions): array
+    {
+        $parameters = [
+            'name' => $name,
+            'permissions' => [
+                'canEditTeamPackages' => (bool) $permissions->canEditTeamPackages,
+                'canAddPackages' => (bool) $permissions->canAddPackages,
+                'canCreateSubrepositories' => (bool) $permissions->canCreateSubrepositories,
+                'canViewVendorCustomers' => (bool) $permissions->canViewVendorCustomers,
+                'canManageVendorCustomers' => (bool) $permissions->canManageVendorCustomers,
+            ],
+        ];
+
+        return $this->put(sprintf('/teams/%s/', $teamId), $parameters);
+    }
+
+    public function remove($teamId): bool
+    {
+        // Returns empty response on success.
+        $this->delete(sprintf('/teams/%s/', $teamId));
+        return true;
```

Comment:
I think this should stay consistent with other DELETE API calls and just return whatever the response returns. On failure this already throws an exception. Same for the call below